dfc/metasync.go

```diff
@@ -116,7 +116,7 @@ func newmetasyncer(p *proxyrunner) (y *metasyncer) {
 	y.chfeedwait = make(chan []interface{})
 
 	// create the ticker, do not need to start at start up
```

Comment:
Seems like this comment is misleading. Remove?

Reply:
will change it
will also change the ticker name, i was debating about it, think of it now, 'timer' is a better name, and it is actually a timer not a ticker
thank you both for the review